### ISSUE TITLE
race condition in HA environment

### ISSUE THREAD

**Issue by bennydubois** [ncabatoff/script-exporter#11]
Having two Prometheus servers scrape script exporter in a HA setup causes a race condition where only one server gets the metrics.

Prometheus version: 2.8
Script exporter version: current build, not running in docker

**Comment by ncabatoff** [comment]
Have you tried increasing `-script-workers` from the default of 1?

**Comment by bennydubois** [comment]
yes, I increased this setting up to 4 but there is still only one of the Prometheus servers that gets metrics at a time.
